README.md

```diff
@@ -6,12 +6,13 @@
 
 - css
 - html
+- Javascript
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@til2to](https://github.com/til2to)
+- GitHub: [@til2to](https://til2to.github.io/porfolio_projects/)
 
```

Comment:
- MIT license link in your ReadMe is broken on _line 35_, kindly endeavor to fix it.
Implementing the above changes will make your ReadMe appear more professional.